mozilla_version/test/test_gecko.py

```diff
@@ -188,6 +187,20 @@ def test_gecko_version_is_of_a_defined_type(version_string, expected_type):
     assert getattr(release, 'is_{}'.format(expected_type))
 
 
+@pytest.mark.parametrize('string, expected', ((
+    '/firefox/nightly/latest-mozilla-central-l10n/firefox-63.0a1.:lang.linux-i686.tar.bz',
```

Comment:
Nit: could we add this test as well? I want to make sure [this tests](https://github.com/mozilla-releng/bouncerscript/blob/master/bouncerscript/test/test_script.py#L123) pass 🤔 

```
(
    '/firefox/nightly/latest-mozilla-central/firefox-63.0a1.en-US.win32.installer.exe'
    '63.0a1',
), (
```

Reply:
Sure! Added.